auditexport/cli.py

```diff
@@ -46,12 +46,18 @@ def main():
                         dest='syslogport',
                         default='514',
                         required=False)
-    parser.add_argument('-b', '--bookmark',
-                        help='Start from last bookmark',
-                        dest='useBookMark',
+    parser.add_argument('-c', '--cursor',
+                        help='Start from last cursor',
+                        dest='useCursor',
                         default=False,
                         required=False,
                         action='store_true')
+    parser.add_argument('-l', '--limit',
+                        help='Number of records we pull for each chunk when we use --cursor. Default is 100',
+                        dest='limit',
+                        default=100,
+                        required=False,
+                        action='store_true')
```

Comment:
![Screen Shot 2019-03-12 at 19 45 21](https://user-images.githubusercontent.com/41641307/54223123-7f9f3400-44ff-11e9-8255-178501a82f93.png)

@Nnamdi101 exception looks strange)) Have any ideas?

Reply:
remove `action='store_true` from limit.  That is used only when you want to specify the param as a true/false param. For example adding `-v` option automatically sets the verbose param. No need to do `-v=true`.